website/content/api-docs/system/secrets-sync.mdx

```diff
@@ -541,3 +582,130 @@ $ curl \
     "auth": null
 }
 ```
+
+## List associations
+
+This endpoint lists all secrets referenced by at least one association. Each secret can be associated with multiple
+destinations, so the total number of associations can be greater than the total number of synced secrets.
+
+| Method | Path                     |
+|:-------|:-------------------------|
+| `LIST` | `/sys/sync/associations` |
+
+### Sample request
+
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request LIST \
+    http://127.0.0.1:8200/v1/sys/sync/associations
+```
+
+### Sample response
+
+```json
+{
+    "request_id": "uuid",
+    "lease_id": "",
+    "renewable": false,
+    "lease_duration": 0,
+    "data": {
+        "key_info": {
+            "my-kv-1/my-secret-1": [
+                {
+                    "mount": "my-kv-1/",
+                    "secret_path": "my-secret-1"
+                }
+            ],
+            "my/kv/1/my/secret/1": [
+                {
+                    "mount": "my/kv/1/",
+                    "secret_path": "my/secret/1"
+                }
+            ]
+        },
+        "keys": [
+            "my/kv/1/my/secret/1",
+            "my-kv-1/my-secret-1"
+        ],
+        "total_associations": 4,
+        "total_secrets": 2
+    },
+    "wrap_info": null,
+    "warnings": null,
+    "auth": null
+}
+```
+
+## Read destinations by secret
+
+This endpoint returns all destinations associated with the provided secret reference.
+
+<Note>
+
+  The request is available with both path parameters or request parameters to support
+  mounts or secrets with forward slash characters (`/`).
```

Comment:
```suggestion
  The request is available with either path parameters or request parameters to support
  mounts or secrets with forward slash characters (`/`).
```
Does "either" make more sense since you should choose one of the methods to provide the values?